test/testlib/nuodb_admin_utilities.go

```diff
@@ -193,10 +206,23 @@ func AwaitNrLoadBalancerPolicies(t *testing.T, namespace string, podName string,
 		return err == nil && len(policies) == expectedNumber
 	}, 30*time.Second)
 }
-
+/*
+	Deprecated
```

Comment:
Nit: [Godoc](https://go.dev/blog/godoc) is looking for string literal `Deprecated:` on a new paragraph.

> Sometimes a struct field, function, type, or even a whole package becomes redundant or unnecessary, but must be kept for compatibility with existing programs. To signal that an identifier should not be used, add a paragraph to its doc comment that begins with “Deprecated:” followed by some information about the deprecation.